src/quo2/components/switchers/group_messaging_card/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.switchers.group-messaging-card.view
+  (:require
+    [react-native.core :as rn]
+    [quo2.components.avatars.group-avatar :as group-avatar]
+    [quo2.components.switchers.base-card.view :as base-card]
+    [quo2.components.switchers.card-main-info.view :as card-main-info]
+    [quo2.components.switchers.card-content.view :as card-content]
+    [quo2.components.switchers.group-messaging-card.style :as style]
+    [quo2.components.switchers.utils :as utils]
+    [quo2.foundations.colors :as colors]))
+
+(defn view
+  [{:keys [avatar type status title customization-color on-close content]}]
+  (let [color-60 (colors/custom-color customization-color 60)]
```

Comment:
I think we should call it `color` or `custom-color` or something like that. `-60` seems more like implementation details than describing what it is